src/sentry/static/sentry/app/components/stackedBarChart.jsx

```diff
@@ -327,20 +327,164 @@ const StackedBarChart = createReactClass({
   },
 
   render() {
-    let {className, style, height, width} = this.props;
-    let figureClass = [className, 'barchart'].join(' ');
+    let {height} = this.props;
     let maxval = this.maxPointValue();
 
     return (
-      <figure className={figureClass} style={{height, width, ...style}}>
-        <span className="max-y">
-          <Count value={maxval} />
-        </span>
-        <span className="min-y">0</span>
-        <span>{this.renderChart()}</span>
-      </figure>
+      <StyledBarChart showAxis={this.props.showAxis} className="barchart">
+        {this.props.showAxis && (
+          <Axis>
+            <AxisMax>
+              <Count value={maxval} />
+            </AxisMax>
+            <AxisMin>0</AxisMin>
+          </Axis>
+        )}
+        <Columns justify="space-between" style={{height, minHeight: '30px'}}>
+          {this.renderChart()}
+        </Columns>
+      </StyledBarChart>
     );
   },
 });
 
+const StyledBarChart = styled.figure`
+  position: relative;
+  display: block;
+  align-items: flex-end;
+  padding: ${p => (p.showAxis ? '5px 0 5px 25px' : 0)};
+`;
+
+const Columns = styled(Flex)`
+  margin-left: -1px;
+  margin-right: -1px;
+`;
+
+const Column = styled.div`
+  height: 100%;
+  text-indent: -9999em;
+  flex: 1;
+  position: relative;
+  border-left: 1px solid transparent;
```

Comment:
you could probably just use `calc` for this? also if you switch to css grid you can do `grid-column-gap: 1px`